Upgrade to new multicast resolver

mcm001 opened the pull request.

mcm001 commented:
I think we can get rid of the old way of finding the roboRIO too? this should always work. This is also a polling arch, which right now only fires on UI events. Maybe we should republish this data periodically?

gerth2 commented:
Agreed on periodic re-check if it's not a huge tear-up.... at least some users will get burned by it.... or not knowing they need to refresh the page before taking a screenshot and posting it for support.

Gold856 commented:
OBE by #444? And also really stale

mcm001 commented:
Not actually OBE. We never did add the mdns responder, but iirc there’s an issue open to track already

Gold856 committed:
Upgrade to 2027 alpha

Gold856 committed:
Switch to Ubuntu 24.04

spacey-sooty committed:
Start on updating examples for 2027

Signed-off-by: Jade Turner <spacey-sooty@proton.me>

Gold856 committed:
Upgrade Gradle, fix build, and format

Gold856 committed:
Disable example builds

jlmcmchl committed:
[2027] Add systemcore as a photonlib build target (#1995)

## Description

Added systemcore to a couple of build files in order for `./gradlew
publishToMavenLocal` to generate systemcore-compatible dependencies.

Needed to support deploying photonlib to systemcore.

## Meta

Merge checklist:
- [x] Pull Request title is [short, imperative
summary](https://cbea.ms/git-commit/) of proposed changes
- [x] The description documents the _what_ and _why_
- [ ] If this PR changes behavior or adds a feature, user documentation
is updated
- [ ] If this PR touches photon-serde, all messages have been
regenerated and hashes have not changed unexpectedly
- [ ] If this PR touches configuration, this is backwards compatible
with settings back to v2024.3.1
- [ ] If this PR touches pipeline settings or anything related to data
exchange, the frontend typing is updated
- [ ] If this PR addresses a bug, a regression test for it is added

---------

Co-authored-by: Gold856 <117957790+Gold856@users.noreply.github.com>

Gold856 committed:
Add DataLog to the list of libraries loaded

jlmcmchl committed:
Upgrade to 2027 alpha 2 (#2010)

Gold856 committed:
Add mrcal dependency packages

mcm001 committed:
Create roborio finder

mcm001 committed:
stuff and things

mcm001 committed:
still does not work?

mcm001 committed:
fix up C++

mcm001 committed:
More things
